File: src/1887.py
class Solution:
    def reductionOperations(self, nums: list[int]) -> int:
        nums.sort()
        n = len(nums)
        ret = 0
        prev = nums[0]
        for i in range(n):
            if nums[i] != prev:
                ret += n - i
                prev = nums[i]
        return ret

File: src/1887.cpp
#include <vector>
#include <algorithm>

using namespace std;

class Solution {
public:
    int reductionOperations(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int ret = 0, prev = nums[0];
        int n = nums.size();
        for (int i = 0; i < n; ++i) {
            if (nums[i] != prev) {
                ret += n - i;
                prev = nums[i];
            }
        }
        return ret;
    }
};
